Signal/src/view controllers/UITests/SignalsViewController.m

```diff
@@ -324,10 +324,8 @@ - (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath
 
 - (void)presentThread:(TSThread *)thread keyboardOnViewAppearing:(BOOL)keyboardOnViewAppearing {
     dispatch_async(dispatch_get_main_queue(), ^{
-      if (!_mvc) {
-          _mvc = [[UIStoryboard storyboardWithName:@"Storyboard" bundle:NULL]
-              instantiateViewControllerWithIdentifier:@"MessagesViewController"];
-      }
+      _mvc = [[UIStoryboard storyboardWithName:@"Storyboard" bundle:NULL]
+                instantiateViewControllerWithIdentifier:@"MessagesViewController"];
```

Comment:
FYI @FredericJacobs, I think we should roll back the "shared MessageViewsController" from this release. We've had too many separate issues as a result of it, and we're not really leveraging for any substantial advantage at this point (e.g. there is no iPad layout).

Did want to give you a chance to comment though.